src/postprocess/chi0.jl

```diff
@@ -117,11 +121,12 @@ Returns the change in density `δρ` for a given `δV`. Drop all non-diagonal te
 (f(εn)-f(εm))/(εn-εm) factor less than `droptol`. If `sternheimer_contribution`
 is false, only compute excitations inside the provided orbitals.
 """
-function apply_χ0(ham, ψ, εF, eigenvalues, δV::RealFourierArray;
-                  droptol=0,
-                  sternheimer_contribution=true,
-                  temperature=ham.basis.model.temperature,
-                  cgtol=1e-6)
+@timing function apply_χ0(ham, ψ, εF, eigenvalues, δV::RealFourierArray;
+                          droptol=0,
+                          sternheimer_contribution=true,
+                          temperature=ham.basis.model.temperature,
+                          cgtol=1e-6,
+                          verbose=false)
```

Comment:
A bit tricky here: it's not clear that the verbosity is in the Sternheimer solve. I would either not expose this, change its name, or have a sternheimer_kwargs argument. In fact we should maybe do this more (having a "X_kwargs" field)